main/adc/adc.h

```diff
@@ -0,0 +1,18 @@
+#ifndef ADC_H_
```

Comment:
Use the default header guard style, two _ infront and behind. Aka;
`__ADC_HEADER__`